src/ClientGenerator/CSharpCodeGenerator.cs

```diff
@@ -394,13 +394,13 @@ protected override void AddGetGrainMethods(GrainInterfaceData iface, CodeTypeDec
                 add(@"
                         public static {0} GetGrain(long primaryKey, string keyExt)
                         {{
-                            return Cast(global::Orleans.CodeGeneration.GrainFactoryBase.MakeKeyExtendedGrainReferenceInternal(typeof({0}), {1}, primaryKey, keyExt));
+                            return Cast(global::Orleans.CodeGeneration.GrainFactoryBase.MakeKeyExtendedGrainReferenceInternal(typeof({0}), primaryKey, keyExt));
```

Comment:
Ohh, sorry, I misread the code.  The {1} applies to the removed CodeSnippetTypeMember interfaceId.
Reviewing code via GitHub is just so bad.